### PR TITLE
completing adding English Comments

### PR DESCRIPTION
**Pull request by ghana7989**
Without Deleting Japanese comments, added English comments below for the ease of future contributors

### CODE REVIEW

File: src/synth/synth.ts [hunk 1]
```diff
@@ -135,7 +138,8 @@ export default class SynthController {
   }
 }
 
-/// メッセージがチャンネルイベントならチャンネルを、そうでなければ -1 を返す
+// メッセージがチャンネルイベントならチャンネルを、そうでなければ -1 を返す
+// Returns channel events to channel events, otherwise -1
```
Comment (ryohey) [inline src/synth/synth.ts:142]: Returns channel number when message is channel events, otherwise -1

**Review by ryohey — CHANGES_REQUESTED** [review]
Thank you for your contribution!
I commented on the details.

File: src/common/track/Track.ts [hunk 2]
```diff
@@ -114,6 +114,7 @@ export default class Track {
   }
 
   // ソート、通知を行わない内部用の addEvent
+  // Sorting, AddEvent for internal notification
```
[the remark this answers is not in the document]
Comment (ghana7989) [reply src/common/track/Track.ts:117]: I was using google translate

File: src/main/components/PianoRollToolbar/QuantizeSelector/QuantizeSelector.tsx [hunk 3]
```diff
@@ -63,12 +63,15 @@ export interface QuantizeSelectorProps {
 
 function QuantizeSelector({ value, onSelect }: QuantizeSelectorProps) {
   // 整数ではなく 1.5 をかけると整数になるとき付点
+  // Attachment to an integer when it is 1.5 instead of integer
   const dot = value % 1 !== 0 && (value * 1.5) % 1 === 0
 
   // 1.5 で割ると整数になるとき3連符
+  // When divided by 1.5, when it becomes an integer 3
```
[the remark this answers is not in the document]
Comment (ghana7989) [reply src/main/components/PianoRollToolbar/QuantizeSelector/QuantizeSelector.tsx:70]: Yeah what you said is correct

**Review by ryohey — APPROVED** [review]
Thank you!

**Pull request merged**